lib/shared/layers/python-sdk/python/genai_core/models.py

```diff
@@ -57,7 +57,8 @@ def list_azure_openai_models():
     # azure openai model are listed, comma separated in
     # AZURE_OPENAI_MODELS variable in external API secret
     models = genai_core.parameters.get_external_api_key("AZURE_OPENAI_MODELS") or ""
-
+    if not models:
+        return []
```

Comment:
I would None to be consistent with the other methods in this file.

Reply:
Updated